DashAISoftware/DashAI
# Move getFullDefaultValues function to /api folder

# Summary


Move `getFullDefaultValues` function to `/api` folder and define types for a configurable object  

## Type of change



- Refactoring.

## Changes


- moved the code of `getFullDefaultValues` function from `/utils` to `/api`
- adapt the code of `getFullDefaultValues` to TypeScrpit (state the correct types for variables/functions)
- fix the imports of `getFullDefaultValues`

## How to Test
1. Go to `DashAI/DashAI/front` and run `yarn build`
2. Go back to `DashAI/` and run `python -c "import DashAI;DashAI.run()"`
3. Navigate to the Experiments tab
4. Click the "New Experiment" button
5. Give a name to your experiment and select "TabularClassification" as your task.
6. Select your dataset.
7. Try to add models, the form render and default values should work properly.



## Notes


- This PR depends on #73
